rdkit/Chem/PandasTools.py

```diff
@@ -801,64 +638,55 @@ def _runDoctests(verbose=None):  # pragma: nocover
     sys.exit(failed)
 
 
+InstallPandasTools()
+
+
 if __name__ == '__main__':  # pragma: nocover
```

Comment:
I don't believe that any of these tests are actually run. In order to make that happen, `PandasTools.py` would have to be added to the `test_list.py`

Reply:
I admit I did not touch that part - I can fix that after the beta is released.